docker-compose.yml

```diff
@@ -26,25 +26,6 @@ services:
     ports:
       - "80:80"
 
-  config-server:
-    build:
-      context: .
-      dockerfile: config-server/Dockerfile
-      args:
-        SVC: config-server
-    environment:
-      SERVER_PORT: "8888"
-      SPRING_CLOUD_CONFIG_SERVER_GIT_URI: "https://github.com/NDPH-ARTS/spring-config-server.git"
-      SPRING_CLOUD_CONFIG_SERVER_GIT_SEARCHPATHS: "practitioner-service"
-    image: ghcr.io/ndph-arts/config-server:${GITHUB_SHA}
-    restart: on-failure
-    ports:
-      - "8888:8888"
-    healthcheck:
-      test: curl http://localhost:8888/
-      interval: 10s
-      timeout: 10s
-
```

Comment:
Can you keep the config server here in case we want to run the system fully in docker-compose?